net-im/swiften/metadata.xml

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
+<pkgmetadata>
+	<maintainer type="person">
+		<email>ck+gentoo@bl4ckb0x.de</email>
+		<name>Conrad Kostecki</name>
+	</maintainer>
+	<maintainer type="project">
+		<email>proxy-maint@gentoo.org</email>
+		<name>Proxy Maintainers</name>
+	</maintainer>
+	<longdescription>
+		Swiften is a robust, high-quality, standards-compliant,
+		cross-platform, and performant C++ library for
+		implementing XMPP applications.
+	</longdescription>
+	<use>
+		<flag name="debug">Compile with debug information.</flag>
+		<flag name="doc">Install optional documentation.</flag>
```

Comment:
Don't redefine flags when global descriptions are no worse than yours. Applies to most below too.

Reply:
I see, will drop them all, as the global use flag description is fine.

Reply:
Dropped now.